docs_src/getting-started/quick-start/index.md

```diff
@@ -1,227 +1,255 @@
-        "responses":[
-            {
-                "code":"200",
-                "description":null,
-                "expectedValues":[
-                "RandomValue_Int8"
-                ]
-            },
-            {
-                "code":"503",
-                "description":"service unavailable",
-                "expectedValues":[
-
-                ]
-            }
-        ],
-        "url":"http://edgex-core-command:48082/api/v1/device/5c0e8a259f8fc20001a5d230/command/5c0e8a259f8fc20001a5d22b"
-    },
-    "put":{
-        "path":"/api/v1/device/{deviceId}/GenerateRandomValue_Int8",
-        "responses":[
-            {
-                "code":"200",
-                "description":null,
-                "expectedValues":[]
-            },
-            {
-                "code":"503",
-                "description":"service unavailable",
-                "expectedValues":[]
-            }
-        ],
-        "parameterNames":[
-            "Min_Int8",
-            "Max_Int8"
-        ],
-        "url":"http://edgex-core-command:48082/api/v1/device/5c0e8a259f8fc20001a5d230/command/5c0e8a259f8fc20001a5d22b"
-    }
+  "created": 1592190157924,
+  "modified": 1592190157924,
+  "id": "5353248d-8006-4b01-8250-a07cb436aeb1",
+  "name": "GenerateRandomValue_Int8",
+  "get": {
+    "path": "/api/v1/device/{deviceId}/GenerateRandomValue_Int8",
+    "responses": [
+     {
+       "code": "200",
+       "expectedValues": [
+          "RandomValue_Int8"
+       ]
+     },
+     {
+       "code": "503",
+       "description": "service unavailable"
+     }
+    ],
+    "url": "http://edgex-core-command:48082/api/v1/device/4a602dc3-afd5-4c76-9d72-de02407e80f8/command/5353248d-8006-4b01-8250-a07cb436aeb1"
+  },
+  "put": {
+    "path": "/api/v1/device/{deviceId}/GenerateRandomValue_Int8",
+    "responses": [
+      {
+        "code": "200"
+      },
+      {
+        "code": "503",
+        "description": "service unavailable"
+      }
+    ],
+    "url": "http://edgex-core-command:48082/api/v1/device/4a602dc3-afd5-4c76-9d72-de02407e80f8/command/5353248d-8006-4b01-8250-a07cb436aeb1",
+    "parameterNames": [
+      "Min_Int8",
+      "Max_Int8"
+    ]
+  }
 }
 ```
-
 !!! Note
     The URLs won't be exactly the same for you, as the generated unique IDs for both the Device and the Command will be different. So be sure to use your values for the following steps.
 
-You'll notice that this one command has both a **get** and a **put** option. The **get** call will return a random number, and is what is being called automatically to send data into the rest of EdgeX. You can also call it manually using the **get** URL provided:
+You'll notice that this one command has both a **get** and a **put** option. The **get** call will return a random number, and is what is being called automatically to send data into the rest of EdgeX (specifically core data). You can also call **get** manually using the URL provided:
```

Comment:
i think capital letters for the HTTP Verbs would be good "GET" "PUT"

Reply:
fixed